util.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __UTIL_H__
```

Comment:
this change does not relate to this commit subject

Reply:
I will update commit message to be more informational.

Reply:
I have updated patch with commit message with all reasons.